content/docs/infrastructure-wg.md

```diff
@@ -0,0 +1,196 @@
+community is appreciated in all of these steps. In parallel to this, community 
+members are welcome to take over certain SOWs and will get help from the 
+respective contractor if needed. If that work touches shared infrastructure or 
+impacts the future maintenance, the solution needs to be agreed with the 
+working group and the contractor.
+
+## Proposal for implementation and operations
+
+To keep the workload for the IWG low and scale up the work, the implementation
+of changes shall be mostly done by a 3rd party. 
+
+*Operations of services* shall be done by contractors with whom we can have
+Service Level Agreements (availibility, response times, recovery time, 
+handling GDPR requests, ...).
+
+For *implementing individual changes* there are a few options:
+* Community volunteers: There is someone from the community willing to 
+  implement the change.
+* Community bounty: We're offering a bounty for a community member to implement
+  the change. This is a chance to give back something to community members
+  who are not paid for their contirbutions by some company or organisation.
+* Contractor: In case something is very large or we can't find anyone to do it
+  the work can be handed off to a contractor.
+
+
+## Ideas for initial work items
+
+This is an unsorted, unprioritized list of ideas the working group could look
+into: 
+
+* Run a survey in the community to understand the largest pain points and 
+  prioritze potential improvements.
+* Create and maintain a public documentation of the existing infrastructure and 
+  the supported workflows (what, why, who), including down-stream usages of 
+  LLVM.
+* Define and agree a vision for LLVM infrastructure and for the design 
+  principles thereof: What should the development workflow and the 
+  infrastructure look like?
+* Identify the need for addons for Github the community is missing right now 
+  (e.g. subscribing to labels on Github Issues).
+* Come to a decision on using Phabricator vs. GitHub Pull Requests for code
+  reviews, then implement the change.
+* Create a roadmap for the handover of the existing infrastructure to a 
+  contractor. Components ready for immediate takeover:
+  * Phabricator (in case we stay with it)
+  * Pre-merge testing
+  * Buildbot workers
+  * clangd remote index for LLVM
+  * [LNT service](http://lnt.llvm.org/)
+  * ...
+* Create new design and structure for the LLVM website, migrate the existing 
+  content there.
+* Create a test strategy for the LLVM project and refactor the existing build 
+  infrastructure to meet the new test strategy. Some ideas:
+  * Multi-stage CI: run “cheap” builds first, run “expensive” builds only on
+    commits where the cheap builds have passed.
+  * Set up a collector dashboard showing build results for all CI systems per 
+    commit, e.g. Treeherder or something GitHub based.
+  * Investigate if we can get a good deal from GitHub Actions with sponsored 
```

Comment:
With my proposal (to be demonstrated in the video to be released) we possibly don't even need extra hardware.

Reply:
can't wait to see that 😄